cli/internal/cache/cache_fs.go

```diff
@@ -38,27 +41,36 @@ func (f *fsCache) Fetch(target, hash string, _unusedOutputGlobs []string) (bool,
 
 func (f *fsCache) Put(target, hash string, duration int, files []string) error {
 	g := new(errgroup.Group)
-	for i, file := range files {
-		_, file := i, file // https://golang.org/doc/faq#closures_and_goroutines
-		hash := hash
+
+	var numDigesters int = runtime.NumCPU()
```

Comment:
Style: this should be `numDigesters := runtime.NumCPU()`.

However, as a more general comment, I suspect we'll want this value to be configurable.

Reply:
@gsoltis any chance we can get this merged without the configurable option? I understand a parameter would be good, but I feel it's best we fix the issue first.